pkg/vcs/github/client.go

```diff
@@ -265,57 +290,69 @@ func (c *Client) CloneMergeRequest(project string, mr vcs.MR, dest string) (vcs.
 	return zgit.NewRepository(gitRepo, auth, dest), nil
 }
 
-func (c *Client) UpdateMergeRequestDiscussionNote(mrIID, noteID int, project, discussionID, comment string) (vcs.MRNote, error) {
+func (c *Client) UpdateMergeRequestDiscussionNote(ctx context.Context, mrIID, noteID int, project, discussionID, comment string) (vcs.MRNote, error) {
 	//TODO implement me
 	//panic("implement me")
 	return nil, nil
 }
 
-func (c *Client) ResolveMergeRequestDiscussion(s string, i int, s2 string) error {
+func (c *Client) ResolveMergeRequestDiscussion(ctx context.Context, s string, i int, s2 string) error {
 	// This is a NoOp on GitHub
 	return nil
 }
 
-func (c *Client) AddMergeRequestDiscussionReply(prID int, fullName, discussionID, comment string) (vcs.MRNote, error) {
+func (c *Client) AddMergeRequestDiscussionReply(ctx context.Context, prID int, fullName, discussionID, comment string) (vcs.MRNote, error) {
+	ctx, span := otel.Tracer("TFC").Start(ctx, "ApplyMergeRequestDiscussionReply")
```

Comment:
Was this meant to be AddMergeRequestDiscussionReply ?

Reply:
Yeah we might have renamed the method good catch